src/Abp.Web.Api/WebApi/Controllers/Dynamic/Selectors/HttpVerbExtensions.cs

```diff
@@ -80,6 +82,11 @@ public static HttpVerb ToHttpVerb(this HttpMethod method)
                 return HttpVerb.Head;
             }
 
+            if (method == new HttpMethod("PATCH"))
```

Comment:
Yes, it is.  I used reflector to find out how it works, but you can also check out [this link](https://searchcode.com/codesearch/view/28600541/). Behavior of the operator '=='. Equals method is quite difference, but it work fine for all of the cases as well.